qiling/os/filestruct.py

```diff
@@ -50,6 +48,12 @@ def close(self):
     def fstat(self):
         return Fstat(self.__fd)
     
+    def fcntl(self, fcntl_cmd, fcntl_arg):
+        try:
+            return fcntl.fcntl(self.__fd, fcntl_cmd, fcntl_arg)
+        except Exception:
+            pass
+
```

Comment:
why try and except. anything we need to worry about ?

Reply:
There is a chance that the module `fcntl` may be imported unsuccessfully, as shown in the begin of the file. So maybe this `try-except` is used for this?

Reply:
i guess is smart for us to remove all of this
i mean try import above and try import here.

Reply:
I try to remove both `try-except` catch, and test failed on windows for no module `fnctl`. There is another `fcntl.ioctl()` in `qiling/os/filestruct.py`, I'm not sure if it's used ot not.  So I try another way: only call `fcntl()` on ql_socket, which is submitted in another pull request.